src/display/process/allProcs.go

```diff
@@ -296,7 +296,7 @@ func AllProcVisuals(dataChannel chan []*proc.Process,
 
 		case data := <-dataChannel:
 			if runAllProc {
-				myPage.ProcTable.CursorColor = selectedStyle
+				//myPage.ProcTable.CursorColor = selectedStyle
```

Comment:
Wait, why was this removed?

Reply:
My bad I didn't save the file before committing.